end_app/weather_app/data_services.py

```diff
@@ -0,0 +1,22 @@
+from .get_weather import get_weather, get_forecast, get_air_pollution
+
+
+class DataView:
+    def __init__(self, city_name):
+        self.city_name = city_name
+        self.weather_data = None
+        self.forecast_data = None
+        self.air_pollution_data = None
+        self.error_message = None
+        self.latitude = None
+        self.longitude = None
+
+        if self.city_name is not None:
+            self.weather_data, self.latitude, self.longitude, self.error_message = get_weather(self.city_name)
+            if self.weather_data is not None:
+                self.forecast_data, error_message = get_forecast(self.latitude, self.longitude)
+                if error_message is not None:
+                    self.error_message = error_message
+                self.air_pollution_data, error_message = get_air_pollution(self.latitude, self.longitude)
+                if error_message is not None:
+                    self.error_message = error_message
```

Comment:
czemu to wszystko dzieje się w inicie? Chyba się trochę nie zrozumieliśmy, mi chodziło, że jak będzie ta klasa, to przenosisz do niej `get_forecast` i ten `get_forecast` będzie jako metoda tej klasy, tak samo `get_air_pollution` i będzie jedna dodatkowa metoda która będzie zwracać wszystkie dane które potrzebujesz, żeby wszystko co dotyczy pobierania danych z API było w jednym miejscu